system/ee/EllisLab/ExpressionEngine/Config/valid_html_form_attributes.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * This source file is part of the open source project
+ * ExpressionEngine (https://expressionengine.com)
+ *
+ * @link      https://expressionengine.com/
+ * @copyright Copyright (c) 2003-2019, EllisLab Corp. (https://ellislab.com)
+ * @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
+ */
+
+/**
+ * HTML Form Attributes
+ * 
+ * Valid HTML attributes that might be specified in a FORM tag 
+ * (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) plus
+ * HTML Global Attributes 
+ * (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes) and
+ * the ARIA Search role attribute
+ * (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role). 
+ * The list last updated on 5 june 2020
+ * 
+ * This list does not include data-* or aria-* as these are wildcards and
+ * need to be handled differently in the code.
+ *
+ */
+return array(
```

Comment:
I'd recommend short array syntax.

There are some line spacing issues too.

Reply:
Uses identical format to captcha.php, mimes.php and stopwords.php in the same folder.
Not sure what the line-spacing issues you refer to are - if you clarify maybe they can be fixed.